src/main/scala/org/broadinstitute/dsde/vault/services/analysis/UpdateServiceHandler.scala

```diff
@@ -64,8 +78,7 @@ case class UpdateServiceHandler(requestContext: RequestContext, dmService: Actor
         dmService ! DmClientService.DMUpdateAnalysis(dmId.get, new AnalysisUpdate(bossObjects))
```

Comment:
This may trigger multiple times.  We need this to check for `bossURLs.size` instead so it only fires once, or use the "Alternatively" logic above.